osu.Game/Overlays/Changelog/ChangelogSupporterPromo.cs

```diff
@@ -138,14 +138,28 @@ private void load(OsuColour colour, TextureStore textures)
                     FillMode = FillMode.Fill,
                     Texture = textures.Get(@"Online/supporter-pippi"),
                 },
-                new Sprite
+                new Container
                 {
                     Anchor = Anchor.TopCentre,
                     Origin = Anchor.TopCentre,
                     Width = 75,
                     Height = 75,
-                    Margin = new MarginPadding { Top = 70 },
-                    Texture = textures.Get(@"Online/supporter-heart"),
+                    Margin = new MarginPadding { Top = 83 },
```

Comment:
Don't change the margin top of the heart.

Move the pippi background slightly above (-28px) to match osu-web. (I was not aware of this when positioning the background).

![Screenshot from 2021-08-15 07-22-31](https://user-images.githubusercontent.com/4964985/129463266-ececc1fc-3789-4d04-9558-51307e0a9e41.png)